src/components/Pagination/Pagination.tsx

```diff
@@ -1 +1,77 @@
-export const Pagination = () => {};
+import cn from 'classnames';
+import { getNumbers } from '../../utils';
+
+interface Props {
+  total: number;
+  perPage: number;
+  currentPage: number;
+  onPageChange: (page: number) => void;
+}
+
+export const Pagination: React.FC<Props> = ({
+  total,
+  perPage,
+  currentPage,
+  onPageChange,
+}) => {
+  const optPages = Math.ceil(total / perPage);
+  const renderPages = getNumbers(1, optPages);
+
+  return (
+    <ul className="pagination">
+      <li
+        className={cn({
+          'page-item': true,
+          disabled: currentPage === 1,
+        })}
+        onClick={
+          currentPage !== 1 ? () => onPageChange(currentPage - 1) : () => {}
+        }
+      >
+        <a
+          data-cy="prevLink"
+          className="page-link"
+          href="#prev"
+          aria-disabled={currentPage === 1 ? `true` : `false`}
+        >
+          «
+        </a>
+      </li>
+      {renderPages.map(page => (
+        <li
+          className={cn({
+            'page-item': true,
+            active: currentPage === page,
+          })}
+          key={page}
+          onClick={() => onPageChange(page)}
+        >
+          <a data-cy="pageLink" className="page-link" href={`#${page}`}>
+            {page}
+          </a>
+        </li>
+      ))}
+
+      <li
+        className={cn({
+          'page-item': true,
+          disabled: currentPage === optPages,
+        })}
+        onClick={
+          currentPage !== optPages
+            ? () => onPageChange(currentPage + 1)
+            : () => {}
+        }
```

Comment:
same here as above. fix it please
Actually better to create handler like
```suggestion
       /// above component
       const handleAddPage = () => {
          if(currentPage !== optPages) {
             onPageChange(currentPage + 1)
           }
       }

        onClick={handleAddPage}
```

Reply:
ideally to do smth like that